[implot] Build error

Failure logs

CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:32 (message):
      
      Failed to download file with error: 1
      If you are using a proxy, please check your proxy setting. Possible causes are:
      
      1. You are actually using an HTTP proxy, but setting HTTPS_PROXY variable
         to `https://address:port`. This is not correct, because `https://` prefix
         claims the proxy is an HTTPS proxy, while your proxy (v2ray, shadowsocksr
         , etc..) is an HTTP proxy. Try setting `http://address:port` to both
         HTTP_PROXY and HTTPS_PROXY instead.
      
      2. If you are using Windows, vcpkg will automatically use your Windows IE Proxy Settings
         set by your proxy software. See https://github.com/microsoft/vcpkg-tool/pull/77
         The value set by your proxy might be wrong, or have same `https://` prefix issue.
      
      3. Your proxy's remote server is out of service.
      
      If you've tried directly download the link, and believe this is not a temporary
      download server failure, please submit an issue at https://github.com/Microsoft/vcpkg/issues
      to report this upstream download server failure.

Call Stack (most recent call first):
  scripts/cmake/vcpkg_download_distfile.cmake:270 (z_vcpkg_download_distfile_show_proxy_and_fail)
  scripts/cmake/vcpkg_from_github.cmake:106 (vcpkg_download_distfile)
  ports/implot/portfile.cmake:3 (vcpkg_from_github)
  scripts/ports.cmake:147 (include)

--- a/ports/implot/portfile.cmake
+++ b/ports/implot/portfile.cmake
@@ -4,7 +4,7 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO epezent/implot
     REF v${VERSION}
-    SHA512 75d5f7e429aaf6eaa3a7eaedf5bcb54ac78fd04121baa8d0fff76bf9a87a52bd27f59bc07df7f81807619f75f570e1f5be70f35a34376f7e7b1f0bf0722b899a
+    SHA512 07e45da79db20f12cd17e3510e0b67085f16865ec27e119cb9faa8037a815c1bc246a9dec11805e031f00a131afbacb6c9ca8a28d1115c4e1365dd941987fc80
     HEAD_REF master
 )
 

--- a/ports/implot/vcpkg.json
+++ b/ports/implot/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "implot",
   "version": "0.15",
+  "port-version": 1,
   "description": "Advanced 2D Plotting for Dear ImGui",
   "homepage": "https://github.com/epezent/implot",
   "license": "MIT",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3390,7 +3390,7 @@
     },
     "implot": {
       "baseline": "0.15",
-      "port-version": 0
+      "port-version": 1
     },
     "indicators": {
       "baseline": "2.3",

--- a/versions/i-/implot.json
+++ b/versions/i-/implot.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "8cd7aca733072549426cfb1e309c4831340713b1",
+      "version": "0.15",
+      "port-version": 1
+    },
+    {
       "git-tree": "67023cbc6fc4301549ef49fa263c70995d693cf5",
       "version": "0.15",
       "port-version": 0

